service/awsconfig/v10/templates/cloudformation/guest/instance.go

```diff
@@ -28,7 +28,7 @@ const Instance = `{{define "instance"}}
       Tags:
       - Key: Name
         Value: {{ .Instance.Master.EtcdVolume.Name }}
-  MountPoint:
+  {{ .Instance.Master.Instance.ResourceName }}MountPoint:
```

Comment:
We also regenerate the volume attachment so CF mounts the etcd volume to the replaced EC2 instance on the stack update.